index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.2/css/bootstrap.min.css">
+</head>
+<body>
+<h1>Student Grade Table</h1>
+
+<table class="table table-striped">
+    <thead class="thead-dark">
+    <tr>
+        <th scope="col">Student Name</th>
+        <th scope="col">Course</th>
+        <th scope="col">Grade</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    </tbody>
+</table>
+
+<script src="https://code.jquery.com/jquery-3.1.0.js"></script>
+<script src="components/app.js"></script>
+<script src="components/grade-table.js"></script>
+<script src="main.js"></script>

```

Comment:
Since `App` will use an instance of `GradeTable`, `app.js` technically depends on `grade-table.js`, even if it doesn't directly reference the `GradeTable` class. Please load other components before loading `app.js`. Since `main.js` is the top-level script that uses everything else, it should still be last.